src/components/Touchable.android.js

```diff
@@ -1,29 +1,49 @@
 // @flow
 import React from "react";
-import { StyleSheet, TouchableNativeFeedback, View } from "react-native";
+import {
+  findNodeHandle,
+  StyleSheet,
+  TouchableNativeFeedback,
+  UIManager,
+  View
+} from "react-native";
 import type { TouchableProps } from "./TouchableTypes";
 import { TouchableDefaultProps } from "./TouchableTypes";
 
-const Touchable = ({
-  children,
-  style,
-  onPress,
-  disabled,
-  ...props
-}: TouchableProps) => (
-  <TouchableNativeFeedback
-    background={TouchableNativeFeedback.SelectableBackground()}
-    onPress={onPress}
-    disabled={disabled}
-    {...props}
-  >
-    <View style={[styles.defaults, style]}>{children}</View>
-  </TouchableNativeFeedback>
-);
+class Touchable extends React.PureComponent<TouchableProps> {
+  static defaultProps = {
+    ...TouchableDefaultProps
+  };
 
-Touchable.defaultProps = {
-  ...TouchableDefaultProps
-};
+  componentDidUpdate(prevProps: TouchableProps) {
+    // Notify user, that the radio button changed to checked state.
+    if (
+      this.props.accessibilityComponentType === "radiobutton_checked" &&
+      this.props.accessibilityComponentType !==
+        prevProps.accessibilityComponentType
+    ) {
+      UIManager.sendAccessibilityEvent(
+        findNodeHandle(this),
+        UIManager.AccessibilityEventTypes.typeViewClicked
```

Comment:
The code for this was taken straight from the [docs](http://facebook.github.io/react-native/docs/accessibility.html#sending-accessibility-events-android).